dataactcore/interfaces/function_bag.py

```diff
@@ -647,3 +647,21 @@ def get_last_validated_date(submission_id):
     # Still need to do a check here in case there aren't any jobs for a submission.
     # This is the case for a single unit test
     return oldest_date.strftime('%m/%d/%Y') if oldest_date else oldest_date
+
+
+def get_action_dates(submission_id):
+    sess = GlobalDB.db().session
+
+    min_action_date = None
+    max_action_date = None
+    date_from_format = '%Y%m%d'
+    date_to_format = '%Y-%m-%d'
+    action_dates = sess.query(func.min(DetachedAwardFinancialAssistance.action_date).label("min_action_date"),
+                              func.max(DetachedAwardFinancialAssistance.action_date).label("max_action_date"))\
+        .filter(DetachedAwardFinancialAssistance.submission_id == submission_id)
+    res = action_dates.one()
+    if res.min_action_date:
+        min_action_date = datetime.strptime(res.min_action_date, date_from_format).date().strftime(date_to_format)
```

Comment:
This may be a dumb question, why are we doing `.date()` in the middle of this? Doesn't `strptime()` create the datetime object we can use `strftime()` on? I could be wrong, I don't use these too often.

Follow-up dumb question. If we only use this to set the reporting start and end dates later in the process, why do we have to format them at all? Does the given variant not work when inserting into a date?